src/utils/styling.js

```diff
@@ -18,8 +18,8 @@ export function getColor(reference, palette, defaultColor) {
 const stylingUtils = {
   cardStyling: ({ Theme, layout }) => {
     const palette = colorUtils.getPalette(Theme);
-    const backgroundColor = getColor(layout.style.backgroundColor, palette, '#e6e6e6');
-    const fontColor = getColor(layout.style.fontColor, palette, 'default');
+    const backgroundColor = getColor(layout.style.background, palette, '#e6e6e6');
+    const fontColor = getColor(layout.style.label, palette, 'default');
```

Comment:
Don't think you can change these only here. Does the styling still works?

Reply:
It was complaining that backgroundColor did not exist on the layout. So looked and they were called background and label. Is that old stuff?

Reply:
Possibly because you were using an older chart. I moved it to backgroundColor and fontColor since it is only containing color info

Reply:
ok I'll reset this